init.lua

```diff
@@ -34,7 +34,7 @@ return require('packer').startup(function()
     requires = { {'nvim-lua/plenary.nvim'} }
   }
   use {
-    "blackCauldron7/surround.nvim",
+    "ur4ltz/surround.nvim,
```

Comment:
You're missing a double-quote at the end of the line

Reply:
Oh yea sorry about that, I'm not on my pc rn but when I get on I'll add it

Reply:
@Infinitybeond1 Do you plan on adding the missing double quote any time soon or should I do it?